packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -461,6 +462,7 @@ export function TableNG(props: TableNGProps) {
                 theme,
                 value,
                 width,
+                timeRange,
```

Comment:
when looking into all of this, I noticed that the `timeRange`, which is passed through to SparklineCell in TableRT, was getting dropped on the floor in TableNG. In practice this doesn't seem to be causing problems but I corrected it here.